pkg/kube/container.go

```diff
@@ -142,6 +142,26 @@ func (c *Container) ExitDescription() string {
 	return c.ociState.ExitDesc
 }
 
+// StateReason returns brief string explaining why container is in its current state.
+// K8s requires us to return CamelCase here, but we will fallback to full description
+// in case of unknown container state.
+func (c *Container) StateReason() string {
+	if c.runtimeState == runtime.StateRunning {
+		// no need for any reason here
+		return ""
+	}
+
+	if c.runtimeState == runtime.StateExited {
+		if c.ExitCode() == 0 {
+			return "Completed"
```

Comment:
constants?

Reply:
Can be, but they will be local to this function, not much difference